nemo_automodel/components/models/deepseek_v3/model.py

```diff
@@ -214,16 +217,12 @@ class DeepseekV3ForCausalLM(nn.Module):
     @classmethod
     def from_config(
         cls,
-        pretrained_model_name_or_path: str | DeepseekV3Config,
+        config: DeepseekV3Config,
```

Comment:
this change will break our current yaml configs for dsv3 and moonlight. can you make the appropriate changes for finetune and pretrain yamls?